src/hooks/useReadyWithDimensions.ts

```diff
@@ -0,0 +1,31 @@
+import {useEffect, useState} from 'react';
+import {Dimensions} from 'react-native';
+import CONST from '@src/CONST';
+
+const useReadyWithDimensions = (isEnabled = true) => {
```

Comment:
Which of ready statement are you planning to introduce for this hook? Please add a comment above the hook to clarify its purpose, or consider renaming the function for better context.